nuxt_src/components/sections/top/events.vue

```diff
@@ -2,18 +2,12 @@
 ## language=yaml
 en:
   event_overview: |
-    It is packed with fun and memorable events such as sponsored exhibits, Japanese food stalls and social gatherings.<br>
-    Please drop in between the sessions.
-  photo_caption1: TEXTTEXTTEXTTEXTTEXT
-  photo_caption2: TEXTTEXTTEXTTEXTTEXT
-  photo_caption3: TEXTTEXTTEXTTEXTTEXT
+    This year's Scala Matsuri will be held online.<br>
+    We are planning to hold many events unique to online, so please look forward to it!
```

Comment:
```suggestion
    We are planning to hold some contents unique to online. Stay tuned!
```

eventというと規模が大きそうな印象なので、contentsぐらいがいいかしら？と思いました！
あとStay tuned!は こうご期待！ みたいなよく使われる軽めの単語なので、今回は温度感あってるかな？と。